drivers/src/soc_ifc.rs

```diff
@@ -74,6 +74,32 @@ impl SocIfc {
         soc_ifc_regs.cptra_security_state().read().debug_locked()
     }
 
+    #[cfg(feature = "runtime")]
```

Comment:
Is this cfg() actually necessary? I doubt this affects the ROM binary. Same below.

Reply:
Yeah looks like it doesn't affect it. Removed.